src/pipe.ts

```diff
@@ -138,12 +142,31 @@ export function createPipe<Ctx extends PipeCtx = PipeCtx<any>>({
     middlewareMap[`${createName}`] = fn || defaultMiddleware;
 
     const tt = req ? (req as any).saga : takeEvery;
+
     function* curSaga(): SagaIterator<void> {
       yield tt(`${action}`, onApi);
     }
     sagas[`${createName}`] = curSaga;
+
     const actionFn = (options?: any) => {
-      const key = encodeBase64(JSON.stringify({ name: createName, options }));
+      const koSort = (opts?: any): object => {
```

Comment:
I'd like to see this function moved to a separate file: `create-key.ts` which also includes the key creation functionality.

Reply:
I felt that this was the better approach as well but I didn't dare to create new files just for this small function.